frontend/main.py

```diff
@@ -131,6 +131,11 @@ def get_hadith(urn: int):
     else:
         return NotFound()
 
+@app.route('/queues/', methods=['GET'])
+@aws_auth.authentication_required
+def get_queue_names():
+    return jsonify({"queue_names":["global","secondary"]})
```

Comment:
Let's return a list of objects and each object having a name property. 
Let's call the method get_queues

We're returning a resource with single property.